### PR TITLE
fix(docs): set 300px width wrapper around textfields

### PR DESCRIPTION
**Pull request by noahcoop**
Issue resolution for https://github.com/dequelabs/cauldron/issues/87

### CODE REVIEW

**Comment by allisonkench** [comment]
Hello Cauldron team! Myself and @noahcoop are contributing to this project as an assignment for UM. Looking forward to hearing your feedback!

File: packages/styles/forms.css [hunk 1]
```diff
@@ -80,6 +80,10 @@ textarea.Field--has-error:focus,
   padding-left: var(--space-half);
 }
 
+.TextFieldContainer {
```
Comment (schne324) [inline packages/styles/forms.css:83]: `TextFieldContainer` is something specific to the demo/docs app and shouldn't be part of the actual `@deque/cauldron-styles` package.  All you need to do is move this css block to 
https://github.com/dequelabs/cauldron/blob/develop/docs/patterns/components/TextField/index.css

**Review by schne324 — CHANGES_REQUESTED** [review]
:+1: there's just one minor thing needing to be addressed

File: packages/styles/forms.css (hunk 1 above)
Comment (noahcoop) [reply packages/styles/forms.css]: Awesome, thanks for the feedback. Moved the css to the correct file in the latest commit

**Comment by schne324** [comment]
👍 thanks for the contribution! The change looks good I just need to figure out why circleci isn't running in this branch. I will get this merged once I sort that out

**Review by schne324 — APPROVED** [review]

**Review by schne324 — CHANGES_REQUESTED** [review]
Sorry for the back-and-forth. I completely overlooked the fact that you updated the actual react component rather than just the demo app. You should update [the TextField documentation/demo component](https://github.com/dequelabs/cauldron/blob/develop/docs/patterns/components/TextField/index.js) instead of  `packages/react/src/components/TextField/index.tsx` because we intentionally leave our inputs at 100% without a max width container to make them work in various contexts.

**Comment by schne324** [comment]
> You should update the TextField documentation/demo component instead of `packages/react/src/components/TextField/index.tsx`

I think all you'll need to do is back out the changes to `packages/react/src/components/TextField/index.tsx` and update the css to:

```css
.TextField form {
  max-width: 300px;
}
```

**Comment by noahcoop** [comment]
Awesome, no worries re the back-and-forth. Just reverted the changes and updated the docs css

**Review by schne324 — APPROVED** [review]

**Comment by schne324** [comment]
# Reviewed for security (gh action still isn't working)

**Pull request merged**